include/dca/phys/dca_step/cluster_solver/ctaux/ctaux_cluster_solver.hpp

```diff
@@ -279,7 +283,8 @@ double CtauxClusterSolver<device_t, parameters_type, Data>::finalize(dca_info_st
 
   dca_info_struct.average_expansion_order(DCA_iteration) = integral / total;
 
-  dca_info_struct.sign(DCA_iteration) = accumulator.get_sign();
+  dca_info_struct.sign(DCA_iteration) =
+      double(accumulator.get_accumulated_sign()) / accumulator.get_number_of_measurements();
```

Comment:
So both of these functions now return int making this double cast necessary.  My question here is whether this is too small for number_of_measurements seems like it should be fine but it did used to be a double.

Reply:
|accumulated_sign| can be at most parameters_.get_measurements(), which is stored as an int. So as long as we do not do more than 2^31-1 measurements (2.2 billions) we are fine. Usually, we do millions to tens of millions of measurements. Otherwise, we have to change a bunch of variables.